src/app/InteractionModelEngine.h

```diff
@@ -213,9 +219,15 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate, public Comman
     CHIP_ERROR OnUnsolicitedReportData(Messaging::ExchangeContext * apExchangeContext, const PayloadHeader & aPayloadHeader,
                                        System::PacketBufferHandle && aPayload);
 
+    void DispatchCommand(CommandHandler & apCommandObj, const ConcreteCommandPath & aCommandPath,
+                         TLV::TLVReader & apPayload) override;
+    bool CommandExists(const ConcreteCommandPath & aCommandPath) override;
+
     Messaging::ExchangeManager * mpExchangeMgr = nullptr;
     InteractionModelDelegate * mpDelegate      = nullptr;
 
+    CommandHandlerInterface * mCommandHandlerList = nullptr;
```

Comment:
Still need to unregister them all on InteractionModelEngine shutdown.  See https://github.com/project-chip/connectedhomeip/pull/11339#discussion_r743413414